app/views/news/index.html.slim

```diff
@@ -0,0 +1,26 @@
+section.content
+  .row
+    .col-md-12
+      .box
+        .box-header.with-border
+          h3.box-title News
+          - if current_user.admin?
+            button.btn.btn-info.btn-sm.pull-right data-target="#newsModal" data-toggle="modal" type="button" Add news
+        /! /.box-header
+        .box-body
+          table.table.table-bordered
+            - @all_news.each do |n|
+              tr
+                td style=("width: 10px")
+                  a href=("/news/#{n.id}")
```

Comment:
Oh, we don't need use helpers here, my fault :(
Maybe `link_to news_path(n) do`